Templates/ScriptCanvasNode/Template/README.md

```diff
@@ -0,0 +1,36 @@
+## How to use this template
+
+Use this template to produce a Script Canvas project that may hold Script Canvas nodes. 
+
+
+This guide will assume that you have created a gem called: `YourGem` using the command: 
+
+```
+scripts\o3de create-gem -gn YourGem -gp Gems\YourGem
+```
+
+From the o3de folder call this command:
+
+```
+scripts\o3de create-from-template -tp Templates\ScriptCanvasNode -dp Gems\YourGem\Code\Source\MyScriptCanvasNodes -r ${GemName} YourGem
+```
+
+## Steps after using this template
+
+After running this template, there are still some manual steps needed in order to make it compile.
+
+### Step 1.
+
+Run the following command to register the newly created Script Canvas nodes project:
+
+```
+scripts\o3de register -esgp Gems\${GemName} -es Gems\${GemName}\Code\Source\${Name}
+```
+
```

Comment:
so this step is going to register the gem and enable it in the project in one command? nice!  Another option would be to update the `scripts\o3de enable-gem` command to register the gem if it isn't already registered

Reply:
Actually in this case, this is NOT registering a gem.
It is registering an "external subdirectory" which is NOT a Gem(FYI the definition of an external subdirectory is a folder with a CMakeLists.txt in it) with the Gem specified at the path of `-esgp/--external-subdirectory-gem-path`

What this does is update the gem.json with the external subdirectory path to the CMake external subdirectory.